SU-SOE/grunt-drupal-tasks
# SOEOP-236: removed brittle menu test

# READY FOR REVIEW

# Summary
- TL;DR - Removed menu test that breaks when client changes IA.

# Needed By (Date)
- soon

# Urgency
- mild

# Steps to Test

1. Follow steps here: https://github.com/SU-SOE/grunt-drupal-tasks/blob/SOE-develop-mc/docs/soe_build_notes.md
2. run command: grunt behat
3. confirm all tests pass.

# Affected Projects or Products
- SOE Main site

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOEOP-236

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
